src/idb_companion.js

```diff
@@ -1,61 +1,65 @@
-const childProcess = require('child_process');
-const debug = require('debug')('testcafe:browser-provider-ios');
-const process = require('process');
+const childProcess = require("child_process");
+const debug = require("debug")("testcafe:browser-provider-ios");
+const process = require("process");
 
 export default {
-    async boot (udid, timeout) {
+    async boot(udid, timeout) {
         try {
-            await this._exec(['--boot', udid], { timeout });
-        }
-        catch (e) {
-            console.error('Failed to boot simulator');
+            await this._exec(["--boot", udid], { timeout });
+        } catch (e) {
+            console.error("Failed to boot simulator");
             console.error(e.stdout.toString());
             process.exit(e.rc);
         }
     },
-    async shutdown (udid, timeout) {
+    async shutdown(udid, timeout) {
         try {
-            await this._exec(['--shutdown', udid], { timeout });
-        }
-        catch (e) { // eslint-disable-line no-empty
-        }
+            await this._exec(["--shutdown", udid], { timeout });
+            // eslint-disable-next-line no-empty, no-unused-vars
+        } catch (e) {}
     },
-    async list () {
+    async list() {
         try {
-            const response = await this._exec(['--list 1']); // eslint-disable-line no-unused-vars
+            const response = await this._exec(["--list 1"]);
 
-            return response.stdout.toString().trim().split('\n');
-        }
-        catch (e) {
+            return response.stdout.toString().trim().split("\n");
+            // eslint-disable-next-line no-unused-vars
+        } catch (e) {
```

Comment:
Just remove `e` instead of disabling the line?

Reply:
The optional catch is not support here. If I removed `(e)`, I'd get build error
```
[10:08:37] SyntaxError in plugin "gulp-babel"
Message:
    /Users/tianfeng/code/testcafe-browser-provider-ios/src/idb_companion.js: Unexpected token, expected ( (26:12)
```
I currently disable the lint rule here and will upgrade dependencies and building tools later.